src/viur/core/modules/user.py

```diff
@@ -1014,7 +1018,7 @@ def authenticator_otp(self, **kwargs):
         if AuthenticatorOTP.verify_otp(otp=otp_token, secret=user["otp_app_secret"]):
             return self._user_module.secondFactorSucceeded(self, user_key)
 
-        skel.errors = [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Wrong OTP Token")]
+        skel.errors = [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Wrong OTP Token", ["otptoken"])]
```

Comment:
The handler `AuthenticatorOTP` doesn't seem to hold a counter for a maximum retry call. Can you please fix this, as it's the case in `TimeBasedOTP`.